src/bindings/python/flux/cli/base.py

```diff
@@ -434,8 +434,32 @@ def update_file(self, path, extension=".toml"):
         self.config = dict_merge(self.config, conf)
         return self
 
+    def _find_config(self, name):
+        # Find a named config as either TOML or JSON in XDG search path
+        for path in util.xdg_searchpath(subdir="config"):
+            # Take the first matching filename preferring TOML:
+            for ext in (".toml", ".json"):
+                filename = f"{path}/{name}{ext}"
+                print(f"checking path {filename}", file=sys.stderr)
```

Comment:
just want to make sure if this is left over debugging?  sort of feels like it could be.

Reply:
Yes, thanks for catching that!